app/[locale]/blog/[slug]/page.tsx

```
@@ -405,17 +390,18 @@
-                      <TableOfContents items={tableOfContents} />
+                      <TableOfContents items={tableOfContents} maxHeight="sm" />
                     </div>
                   </details>
                 </div>
               )}
 
-              <div className="prose prose-lg max-w-none prose-headings:text-secondary-900 dark:prose-headings:text-secondary-100 prose-p:text-secondary-700 dark:prose-p:text-secondary-300 prose-a:text-primary-600 dark:prose-a:text-primary-400 prose-code:text-primary-600 dark:prose-code:text-primary-400 prose-pre:bg-secondary-50 dark:prose-pre:bg-secondary-800 prose-headings:scroll-mt-24">
+              <div className="blog-content prose prose-lg max-w-none">
                 <div
-                  dangerouslySetInnerHTML={{ __html: sanitizedContent }}
+                  dangerouslySetInnerHTML={{ __html: contentWithIds }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)